package.json

```diff
@@ -14,22 +14,22 @@
     "write-heading-ids": "docusaurus write-heading-ids"
   },
   "dependencies": {
-    "@docusaurus/core": "2.0.1",
-    "@docusaurus/plugin-content-blog": "^2.0.1",
-    "@docusaurus/plugin-content-docs": "^2.0.1",
-    "@docusaurus/plugin-content-pages": "^2.0.1",
-    "@docusaurus/plugin-sitemap": "^2.0.1",
-    "@docusaurus/preset-classic": "2.0.1",
-    "@emotion/react": "^11.10.0",
-    "@emotion/styled": "^11.10.0",
+    "@docusaurus/core": "2.1.0",
```

Comment:
I have a question.
After we bumped the module version in `package.json`, the contents in `package-lock.json` will be auto-generated by `npm install`, right?

Reply:
That's great! But sadly, I can't run `npm install` on my side.